Postman Clone

Harikaraja opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to project-guidance here: https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Write your answer here.) YES

### Description

# POSTMAN CLONE

Post Master clone is similar to postman API platform where we can enter the URL and get the information in the format specified by the user.

This project is done using HML,CSS,JS,Bootstrap.

1. we need to choose the request type either GET OR POST.

2. Now we need to specify the content type either JSON or Custom parameters.

3. we can add same many custom parameters we want by clicking on + button.

4. The output will be displayed in the ouput box.

Fixes #858 

<!-- Replace `issue_no` with the issue number which is fixed in this PR -->

### Checklist



- [y] I've read the contribution guidelines.
- [y] I've checked the issue list before deciding what to submit.
- [y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

Kushal997-das reviewed (changes requested):
@Harikaraja You need to change following things : 

1. You need to reduce the image height and width in `readme.md` section .
2. You need to add Deploy link in `readme.md` file .
3. You need to add this project under Webdevelopment `readme.md` file with alphabetical order .

Harikaraja commented:
What is deploy link sir should i host this project using fire base

On Wed, 22 Feb 2023, 21:56 Kushal Das, ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> @Harikaraja <https://github.com/Harikaraja> You need to change following
> things :
>
>    1. You need to reduce the image height and width in readme.md section .
>    2. You need to add Deploy link in readme.md file .
>    3. You need to add this project under Webdevelopment readme.md file
>    with alphabetical order .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kushal997-das/Project-Guidance/pull/859#pullrequestreview-1309634324>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCQWHWTKEVN3QMGNCVZJ6DWYY437ANCNFSM6AAAAAAVCGUVME>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

Kushal997-das reviewed (changes requested):
@Harikaraja You haven't reduce the width and height of the images that are present in `readme.md` file also change the URL this [https://github.com/Harikaraja/Project-Guidance/tree/postman/Web%20Development] to this [https://github.com/Kushal997-das/Project-Guidance/tree/postman/Web%20Development] iwhere you have added the link for the project

Harikaraja commented:
Sir actually I have removed the earlier images and added new images after
cropping how to reduce the width and height sir . Please let me know .

On Fri, 24 Feb 2023, 22:16 Kushal Das, ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> @Harikaraja <https://github.com/Harikaraja> You haven't reduce the width
> and height of the images that are present in readme.md file also change
> the URL this [
> https://github.com/Harikaraja/Project-Guidance/tree/postman/Web%20Development]
> to this [
> https://github.com/Kushal997-das/Project-Guidance/tree/postman/Web%20Development]
> iwhere you have added the link for the project
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kushal997-das/Project-Guidance/pull/859#pullrequestreview-1313892717>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCQWHTHLMENJ3KK74HB7ULWZDQU5ANCNFSM6AAAAAAVCGUVME>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Kushal997-das commented:
> Sir actually I have removed the earlier images and added new images after cropping how to reduce the width and height sir . Please let me know .
> […](#)
> On Fri, 24 Feb 2023, 22:16 Kushal Das, ***@***.***> wrote: ***@***.**** requested changes on this pull request. @Harikaraja <https://github.com/Harikaraja> You haven't reduce the width and height of the images that are present in readme.md file also change the URL this [ https://github.com/Harikaraja/Project-Guidance/tree/postman/Web%20Development] to this [ https://github.com/Kushal997-das/Project-Guidance/tree/postman/Web%20Development] iwhere you have added the link for the project — Reply to this email directly, view it on GitHub <[#859 (review)](https://github.com/Kushal997-das/Project-Guidance/pull/859#pullrequestreview-1313892717)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATCQWHTHLMENJ3KK74HB7ULWZDQU5ANCNFSM6AAAAAAVCGUVME> . You are receiving this because you were mentioned.Message ID: ***@***.***>

No need to crop the image just use width and height size in image URL.

`<img src= "your image source "   width="300" height="300">`

Harikaraja commented:
Should I do this in readme.md file sir

On Fri, 24 Feb 2023, 22:37 Kushal Das, ***@***.***> wrote:

> Sir actually I have removed the earlier images and added new images after
> cropping how to reduce the width and height sir . Please let me know .
> … <#m_3884350595168469596_>
> On Fri, 24 Feb 2023, 22:16 Kushal Das, *@*.*> wrote: @.** requested
> changes on this pull request. @Harikaraja <https://github.com/Harikaraja>
> https://github.com/Harikaraja You haven't reduce the width and height of
> the images that are present in readme.md file also change the URL this [
> https://github.com/Harikaraja/Project-Guidance/tree/postman/Web%20Development]
> to this [
> https://github.com/Kushal997-das/Project-Guidance/tree/postman/Web%20Development]
> iwhere you have added the link for the project — Reply to this email
> directly, view it on GitHub <#859 (review)
> <https://github.com/Kushal997-das/Project-Guidance/pull/859#pullrequestreview-1313892717>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/ATCQWHTHLMENJ3KK74HB7ULWZDQU5ANCNFSM6AAAAAAVCGUVME
> . You are receiving this because you were mentioned.Message ID: *@*.***>
>
> No need to crop the image just use width and height size in image URL.
>
> <img src= "your image source " width="300" height="300">
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kushal997-das/Project-Guidance/pull/859#issuecomment-1444045210>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCQWHWSHWI7US2WCYYKIGDWZDTEJANCNFSM6AAAAAAVCGUVME>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Kushal997-das reviewed (changes requested):
@Harikaraja Fix the things that i have given below

Kushal997-das commented:
![fixthis](https://user-images.githubusercontent.com/61356005/221367935-069054b1-feb6-4b2d-a69b-72dbe9173f8c.JPG)

Harikaraja commented:
Sir should i add my project here ..
Where can I find this file for updating

On Sat, 25 Feb 2023, 21:53 Kushal Das, ***@***.***> wrote:

> [image: fixthis]
> <https://user-images.githubusercontent.com/61356005/221367935-069054b1-feb6-4b2d-a69b-72dbe9173f8c.JPG>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kushal997-das/Project-Guidance/pull/859#issuecomment-1445152491>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCQWHQJT44AT5ZHS4X2K7LWZIWZLANCNFSM6AAAAAAVCGUVME>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Kushal997-das commented:
@Harikaraja In readme.md file you have done some mess just fix this up

Harikaraja commented:
Ok sir

On Sun, 26 Feb 2023, 08:45 Kushal Das, ***@***.***> wrote:

> @Harikaraja <https://github.com/Harikaraja> In readme.md file you have
> done some mess just fix this up
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kushal997-das/Project-Guidance/pull/859#issuecomment-1445257791>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCQWHVF4I2YOU25HHAZVWLWZLDEJANCNFSM6AAAAAAVCGUVME>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Harikaraja commented:
Sir fixed the issue please check sir.
Thank you

Kushal997-das reviewed (approved):
@Harikaraja Great work !
